src/webui/src/components/trial-detail/TableList.tsx

```diff
@@ -458,7 +502,14 @@ class TableList extends React.Component<TableListProps, TableListState> {
                         className="titleColumn"
                     />
                 </Modal>
+                {/* compare trials based message */}
                 <Compare compareRows={selectRows} visible={isShowCompareModal} cancelFunc={this.hideCompareModal} />
+                {/* clone trial parameters and could submit a customized trial */}
+                <Customize
+                    visible={isShowCustomizedModal}
```

Comment:
Suggest move this out. Only render this element when it's visible.

Reply:
Ok.

Reply:
> Suggest move this out. Only render this element when it's visible.

We can't see the `succeed modal` if `<Customize>` component's status is hidden. So...